content/ADR-250-minimap-standard.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: adr
+adr: 250
+title: Explorer Minimap Standard
+date: 2024-05-13
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+- kuruk-mm
+- leanmendoza
+---
+
+# Abstract
+
+This ADR proposes standardizing how we consume the Minimap data and how the Explorers should render it. The objective is to have a standard way of doing that for each Realm and World, not just for Genesis City.
+
+# Context
+
+At the date of publishing this ADR, the Explorers have a Minimap specific to Genesis City. The Expansion of Decentraland created new realms and worlds that were not in Genesis City. More general support requires a common way of displaying the map in the Explorers, including metadata for detailing where the districts are, empty parcels, owned lands, and more.
+
+# Proposal
+
+This proposal maintains compatibility with the current Explorers Minimaps implementation. Also, at the time this is proposed, no known explorer is using the current parameters in the `/about` endpoint.
+
+So, to show the Minimap's data, the protocol for the [/about](https://decentraland.github.io/catalyst-api-specs/#tag/Global/operation/getAboutCatalystInfo) is modified [here](https://github.com/decentraland/protocol/blob/main/proto/decentraland/realm/about.proto), and the `MinimapConfiguration` is modified, making the old parameters deprecated. The new `MinimapConfiguration` is:
+
+```proto
+message MinimapConfiguration {
+  bool enabled = 1;
+
+  // deprecated parameters
+  reserved 2;
+  reserved 3;
+  
+  repeated decentraland.common.BorderRect sizes = 4; 
+  message ImageViewWithZoomLevel {
+    string version = 1; 
+    optional string base_url = 2;
+    optional string suffix_url = 3;
+ }
+  
+  message ParcelView {
+    string version = 1; 
+    optional string image_url = 2;
+ }
+
+  optional ImageViewWithZoomLevel satellite_view = 5;
+  optional ParcelView parcel_view = 6;
+  optional ImageViewWithZoomLevel thumbnail_view = 7;
+}
+
+```
+
+Example for genesis city:
+```json
+{
+  "sizes": [
+ { "left": -150, "top": -150, "right": 150, "bottom": 150 },
+ { "left": 62, "top": 151, "right": 162, "bottom": 158 },
+ { "left": 151, "top": 59, "right": 163, "bottom": 150 }
+ ],
+  "satelliteView": {
+    "version": "v1",
+    "baseUrl": "https://genesis.city/map/latest",
+    "suffixUrl": ".jpg"
+ },
+  "parcelView": {
+    "version": "v1",
+    "imageUrl": "https://api.decentraland.org/v1/minimap.png"
+ }
+}
+```
+
+
+## size
+- The union of all rectangles here represents the places where can have scenes
+- All the other parcels are considered empty; it is up to the explorer to decide if they are walkable or not
+
+With these parameters we can compute an important square, which is the contained one, and the following vertices determine it:
+  - top-left = (minX, maxY) 
+  - bottom-left = (minX, minX) 
+  - top-right = (maxX, maxY) 
+  - bottom-right = (maxX, maxY) 
```

Comment:
```suggestion
  - bottom-right = (maxX, minY) 
```

Reply:
I left a note comment, it's a different coordinate system